scripts/ec2_stage_arm.sh

```diff
@@ -162,6 +161,45 @@ echo "${userdata}"
 echo "======================================================"
 echo
 
+# Install jq
```

Comment:
Future reference: is We shouldn’t need to do this.

Just do —query and —output text